GoogleForCreators/web-stories-wp
# Layers Panel: Remove blue line

## Context

Bug Bash 1.14:

>When you drag the layers panel to expand it or shrink it and you are done dragging it, the blue focus outline sticks around, when you're not dragging anymore that blue outline should go away.

## Summary

Removing the blue line entirely.

## Relevant Technical Choices

The layers panel is not keyboard accessible and hidden from keyboard, so the focus styling is not needed.

## To-do

n/a

## User-facing changes

|Before|After|
|--|--|
|![before](https://user-images.githubusercontent.com/22185279/139733001-cde17a61-e3f4-41d2-a38a-c1ed7d34e78e.gif)|![after](https://user-images.githubusercontent.com/22185279/139733016-e96fbcbc-d5c0-43d3-b881-a84b0643ce35.gif)
|![before-saved](https://user-images.githubusercontent.com/22185279/139733043-4350e551-1382-4c3d-8d24-0d8e74035ec3.gif)|
|no-change|


## Testing Instructions






This PR can be tested by following these steps:

1. Click and drag layer panel - shouldn't see the blue line
2. Clicking the saved preset panels should show the blue line. Clicking and dragging should show the blue line. Keyboard interaction should show the blue line too.

## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #9561
